app/cdap/services/helpers.js

```diff
@@ -506,6 +506,10 @@ function extractErrorMessage(errObj) {
   return objectQuery(errObj, 'response', 'message') || objectQuery(errObj, 'response');
 }
 
+function isUnknownDatabaseError(err) {
```

Comment:
for new exported functions we do jsdoc style comments like https://github.com/cdapio/cdap-ui/blob/62c7fbe190e5f0055454feb94e85c021f35dcaad/app/cdap/services/react/customHooks/useFeatureFlag.ts#L34